.vscode/launch.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
This file should not be committed.

Reply:
Removed this file, and the .vscode folder from source control